localstack/services/route53resolver/models.py

```diff
@@ -106,3 +114,72 @@ def delete_firewall_rule(firewall_rule_group_id, firewall_domain_list_id):
     firewall_rule = get_firewall_rule(firewall_rule_group_id, firewall_domain_list_id)
     region_details.firewall_rules.get(firewall_rule_group_id, {}).pop(firewall_domain_list_id)
     return firewall_rule
+
+
+def get_resolver_query_log_config(id):
+    """returns resolver query log config with the given id if it exists"""
+    region_details = Route53ResolverBackend.get()
+    resolver_query_log_config = region_details.resolver_query_log_configs.get(id)
+    if not resolver_query_log_config:
+        raise ResourceNotFoundException(
+            f"[RSLVR-01601] The specified query logging configuration doesn't exist. Trace Id: '{aws_stack.get_trace_id()}'"
+        )
+    return resolver_query_log_config
+
+
+def delete_resolver_query_log_config(id):
```

Comment:
nit: This method can simply call `get_resolver_query_log_config(id)` to assert existence of the query log config with the given id - the `ResourceNotFoundException` will be raised from there directly. This would help us reduce the duplicate checks and exceptions a bit.

Reply:
Yep. Updated.